OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -174,6 +174,12 @@ public Source(WmlDocument source, int start, int count, string insertId)
         }
     }
 
+    public class DocumentBuilderSettings
+    {
+        public List<string> CustomXmlGuidList = null;
+        public DirectoryInfo DirectoryForDebugOutput = null;
```

Comment:
I definitely see the value in having the ability to output at various stages. My point was abstracting the output so that it could be modified orthogonally to the the processing. By tying it to a directory you limit how you can process those output files. If it was purely for testing while developing it, it may be better to just remove it

Reply:
Yes, good idea, I removed it.